Sources/Orbit/Support/Views/Tab.swift

```diff
@@ -0,0 +1,126 @@
+import SwiftUI
+
+public enum TabStyle {
+    case `default`
+    case product
+    case underlined(Color)
+    case underlinedGradient(Orbit.Gradient)
+
+    public var textColor: Color {
+        switch self {
+            case .default:                            return .inkNormal
+            case .product:                              return .inkNormal
+            case .underlined(let color):                return color
+            case .underlinedGradient(let gradient):     return gradient.color
+        }
+    }
+
+    public var startColor: Color {
+        switch self {
+            case .default:                              return .white
+            case .product:                              return .productNormal
+            case .underlined(let color):                return color
+            case .underlinedGradient(let gradient):     return gradient.startColor
+        }
+    }
+
+    public var endColor: Color {
+        switch self {
+            case .default:                              return .white
+            case .product:                              return .productNormal
+            case .underlined(let color):                return color
+            case .underlinedGradient(let gradient):     return gradient.endColor
+        }
+    }
+}
+
+/// An Orbit sub-component for constructing ``Tabs`` component.
+///
+/// - Important: The Tab can only be used as a part of ``Tabs`` component, not standalone.
+public struct Tab: View {
+
+    let label: String
+    let style: TabStyle
+
+    public var body: some View {
+        // FIXME: Convert to Button with .title4 style for a background touch feedback
```

Comment:
When will this be fixed?

Reply:
I will create a follow-up, as it will not be as straightforward as the note seems.